### DISCUSSION TITLE
Scrypted Plugin

### DISCUSSION THREAD

**Discussion by kylewhirl** [homebridge-simplisafe3/homebridge-simplisafe3#442] [Feature Requests]
Is there any interest in a Scrypted plugin that can display SimpliSafe cameras? Would be nice to have HKSV support through scrypted or just the native plugin. 

https://www.scrypted.app/

**Comment by shamoon** [comment]
Maybe but Scripted has nothing to do with this plugin?

**Reply by kylewhirl** [reply]
You can create plugins for scrypted. I.e. this plugin could probably be built pretty easily within Scrypted

**Comment by kylewhirl** [comment]
Just curious if the plugin dev had any interest in building a SimpliSafe plugin for scrypted, since this is the only one I’ve seen with camera support

**Reply by shamoon** [reply]
Ah, no I’m not, thanks

**Reply by kylewhirl** [reply]
You’re welcome?
